server/game/cards/02_SHD/units/OutlawCorona.ts

```diff
@@ -0,0 +1,20 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+
+export default class OutlawCorona extends NonLeaderUnitCard {
+    protected override getImplementationId () {
+        return {
+            id: '3503780024',
+            internalName: 'outlaw-corona'
+        };
+    }
+
+    public override setupCardAbilities () {
+        this.addBountyAbility({
+            title: 'Put the top card of your deck into play as a resource.',
+            immediateEffect: AbilityHelper.immediateEffects.resourceCard((context) => ({ target: context.player.getTopCardOfDeck() }))
```

Comment:
Let's use `context.source.controller` instead of `context.player`, I'm not 100% sure they're interchangeable in all cases and we know the former works correctly

Reply:
Hey @AMMayberry1, this is a bounty so it doesn't goes to the card controller, but I'll keep that in mind for future implementations!